### ISSUE TITLE
Caching of installation files

### ISSUE THREAD

**Issue by LarsNielsenEG** [potatoqualitee/mssqlsuite#24]
We would like to cache the installation files on windows to speed up the build and save precious minutes :)

I think the fastest way would be to change the download directory from c:\temp to c:\temp\actions\mssqlsuite or any other unique location.

That way we can add a actions/cache step before and get the desired effect. If we cache c:\temp we will cache a lot of unrelated files which is not desirable.

Note: This action could alternatively be changed to internally use a cache if a key was provided (just like https://github.com/actions/setup-node/blob/main/src/cache-save.ts) - But this would also require a greater effort

**Comment by asgerhallas** [comment]
I'd really like this possibility too! 👍

**Comment by mwaddell** [comment]
I played around with caching a bit and it seems like it doesn't help -- https://github.com/mwaddell/mssqlsuite/pull/2

**Comment by potatoqualitee** [comment]
Yep, I do wish this was possible. When I tried, it made it worse or no difference, like @mwaddell's attempts. We'll have to rely on GitHub for this, closing for now.
